apps/server/server.cpp

```diff
@@ -404,17 +404,19 @@ void invoke_sdk_api(payload::ClientRequest buff_recv) {
     }
 
     case GET_AVAILABLE_FRAME_TYPES: {
-        std::vector<aditof::DepthSensorFrameType> aditofFrameTypes;
+        std::vector<std::string> aditofFrameTypes;
+        aditof::DepthSensorFrameType frameDetails;
 
         aditof::Status status =
             camDepthSensor->getAvailableFrameTypes(aditofFrameTypes);
         for (const auto &aditofType : aditofFrameTypes) {
+            camDepthSensor->getFrameTypeDetails(aditofType, frameDetails);
```

Comment:
You need to implement a different use-case in the server for this function as well. In this case only camDepthSensor->getAvailableFrameTypes(aditofFrameTypes) should be used.

Reply:
Added case for GET_FRAME_TYPE_DETAILS